guides/release/tutorial/autocomplete-component.md

```diff
@@ -79,25 +79,25 @@ The `handleFilterEntry` action will apply the search term filter to the list of
 
 ```javascript {data-filename=app/components/list-filter.js}
 import Component from '@ember/component';
+import { action } from '@ember-decorators/object';
+import { classNames } from '@ember-decorators/component';
```

Comment:
`ember-decorators` is not an official Ember addon, so we aren't going to have it in any examples. Ember itself _does_ support the following decorators:

```js
import { computed, action } from '@ember/object';
import { inject as service } from '@ember/service';
import { inject as controller } from '@ember/controller';
import * from '@ember/object/computed';
```

This does mean that we don't have the proper decorators to convert classic components to native class syntax, which is what the core team has agreed on. Everything except class components should always be native classes, and classic components should use classic class syntax.

Reply:
Thanks, all. I'll have a look later today and see what I can fix up.